corehq/apps/cloudcare/static/cloudcare/js/formplayer/middleware.js

```diff
@@ -28,10 +28,24 @@ FormplayerFrontend.module("SessionNavigate", function (SessionNavigate, Formplay
     var clearFormMiddleware = function(name) {
         FormplayerFrontend.trigger("clearForm");
     };
+    var setScrollableMaxHeight = function() {
+        var maxHeight,
+            maxHeightForm;
+        maxHeight = ($(window).height() -
+            FormplayerFrontend.regions.phoneModeNavigation.$el.height() -
+            FormplayerFrontend.regions.breadcrumb.$el.height());
+        maxHeightForm = ($(window).height() -
+            FormplayerFrontend.regions.phoneModeNavigation.$el.height());
+
+        $('.scrollable-container').css('max-height', maxHeight + 'px');
+        $('.form-scrollable-container').css('min-height', maxHeightForm + 'px');
+        $('.form-scrollable-container').css('max-height', maxHeightForm + 'px');
```

Comment:
@benrudolph is it better to have repeating DOM fetch as an object?
```
var scrollable_container = $('.form-scrollable-container')
scrollable_container.css(....)
scrollable_container.css(....)
```
Or chaining in such case is also a good way out?